warning fix request

Warnings in segyio-catr.c

```
segyio-catr.c: In function ‘options parse_options(int, char**)’:
segyio-catr.c:587:12: warning: jump to case label [-fpermissive]
            default: opts.help = 1; opts.errmsg = ""; return opts;
            ^~~~~~~
segyio-catr.c:531:24: note:   crosses initialization of ‘range* r’
                 range* r = opts.r + opts.rsize;
```
Can someone please fix these? Thanks.

--- a/applications/segyio-catr.c
+++ b/applications/segyio-catr.c
@@ -517,7 +517,7 @@ static struct options parse_options( int argc, char** argv ){
                      break;
 
            case 'r': // intentional fallthrough
-           case 't':
+           case 't': {
                 if( opts.rsize == rallocsize ) {
                     rallocsize *= 2;
                     range* re = realloc( opts.r, rallocsize*sizeof( range ) );
@@ -583,6 +583,7 @@ static struct options parse_options( int argc, char** argv ){
                 }
 
                 done: ++opts.rsize; break;
+            }
 
            default: opts.help = 1; opts.errmsg = ""; return opts;
        }

